:100644 100644 4e82028 0000000 M	include/astrea/math/interp1d_linear.h

diff --git a/include/astrea/math/interp1d_linear.h b/include/astrea/math/interp1d_linear.h
index 4e82028..12b0d9e 100644
--- a/include/astrea/math/interp1d_linear.h
+++ b/include/astrea/math/interp1d_linear.h
@@ -27,7 +27,7 @@ inline double f(
   const Eigen::VectorXd& y_Y,
   const double& x
 ) {
-  if (x < x_Y(0) || x > x_Y(Eigen::last)) {
+  if (x < x_Y(0) || x >= x_Y(Eigen::last)) {
     return 0.0;
   }
 
