:100644 100644 75fdcec 0000000 M	include/astrea/element/h_i/oscillator_strengths_stark.h

diff --git a/include/astrea/element/h_i/oscillator_strengths_stark.h b/include/astrea/element/h_i/oscillator_strengths_stark.h
index 75fdcec..da17915 100644
--- a/include/astrea/element/h_i/oscillator_strengths_stark.h
+++ b/include/astrea/element/h_i/oscillator_strengths_stark.h
@@ -13,7 +13,7 @@
 #include "astrea/element/h_i/structure.h"
 
 
-namespace astrea::element::h_i {
+namespace astrea::element::h_i::oscillator_strengths_stark {
 
 
 /**
