:100644 100644 a2ccb8d 0000000 M	include/astrea/element/h_i/oscillator_strengths_doppler.h

diff --git a/include/astrea/element/h_i/oscillator_strengths_doppler.h b/include/astrea/element/h_i/oscillator_strengths_doppler.h
index a2ccb8d..df8667a 100644
--- a/include/astrea/element/h_i/oscillator_strengths_doppler.h
+++ b/include/astrea/element/h_i/oscillator_strengths_doppler.h
@@ -13,7 +13,7 @@
 #include "astrea/element/h_i/structure.h"
 
 
-namespace astrea::element::h_i {
+namespace astrea::element::h_i::oscillator_strengths_doppler {
 
 
 /**
